MultiThreading.Task2.Chaining/Program.cs

```diff
@@ -6,11 +6,18 @@
  * Fourth Task – calculates the average value. All this tasks should print the values to console.
  */
 using System;
+using System.Linq;
+using System.Threading.Tasks;
 
 namespace MultiThreading.Task2.Chaining
 {
     class Program
     {
+        const int MaxQuantity = 10;
+        const int RangeOfNumbers = 100;
+
+        protected static Random RandomGenerator = new Random();
```

Comment:
`protected` access modifier can be used in classes that are intended to be inherited, but Program is usually not inherited. Fields usually use `private` modifier

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/basic-concepts#752-declared-accessibility